providers/base/bin/wifi_nmcli_test.py

```diff
@@ -351,29 +407,49 @@ def print_journal_entries(start):
 
     args = parser.parse_args()
 
-    start_time = datetime.datetime.now()
+    return args
 
-    cleanup_nm_connections()
+
+def main():
+    args = parser_args()
+    start_time = datetime.datetime.now()
     device_rescan()
-    count = list_aps(args)
+    if hasattr(args, "essid"):
+        print("Scanning for APs with ESSID: {}".format(args.essid))
+        aps_dict = list_aps(args.device, args.essid)
+    else:
+        print("Scanning for all APs")
+        aps_dict = list_aps(args.device)
```

Comment:
```suggestion
    aps_dict = list_aps(args.device, args.essid)
```
If I understanding is correct, we did not scanning any wireless network device here. It's already been done in `device_rescan` function.
So I would suggest we do following changes as following
1. assign a default value `None` to the essid parameter
2. change print_head string in the list_aps function. e.g. `List APs with ESSID: Canonical` or `List all APs`

Reply:
Can't do like this, and it will show there is no this attribute error  when execute the script w/o input the essid parameter.

Reply:
After offline discussing, switch another way to do this ~~